frontend/package-lock.json

```diff
@@ -89,112 +89,105 @@
       }
     },
     "node_modules/@angular-devkit/architect": {
-      "version": "0.1703.11",
-      "resolved": "https://registry.npmjs.org/@angular-devkit/architect/-/architect-0.1703.11.tgz",
-      "integrity": "sha512-YNasVZk4rYdcM6M+KRH8PUBhVyJfqzUYLpO98GgRokW+taIDgifckSlmfDZzQRbw45qiwei1IKCLqcpC8nM5Tw==",
+      "version": "0.1901.4",
+      "resolved": "https://cli-467155500999.d.codeartifact.us-east-1.amazonaws.com/npm/cli-npm-artifacts/@angular-devkit/architect/-/architect-0.1901.4.tgz",
```

Comment:
It should be from npm registry

Reply:
That's a good point. We may have to get permission to keep the deps there, but it certainly makes sense in an open-source repo.

Reply:
Yep. Reverting this as per discussion in SRE meeting.

Reply:
I am able to test it locally